pkg/costmodel/router.go

```diff
@@ -1223,11 +1227,13 @@ func Initialize(router *httprouter.Router, additionalConfigWatchers ...*watcher.
 	}
 
 	api := prometheusAPI.NewAPI(promCli)
-	_, err = api.Buildinfo(context.Background())
+	result, err := api.Buildinfo(context.Background())
```

Comment:
I found this API call we're already making when initializing cost-model. The response from this call will have a Prometheus version. I've saved that to a package-wide variable.

Reply:
very nice